app/src/main/java/com/example/parkingsystem/mvp/model/ParkingModel.java

```diff
@@ -0,0 +1,22 @@
+package com.example.parkingsystem.mvp.model;
+
+import java.util.ArrayList;
+
+public class ParkingModel {
+
+    private int parkingSize;
+    private ArrayList<ParkingLot> parkingLots;
+
+    public ParkingModel(){
+        this.parkingLots = new ArrayList<>();
+    }
+
+    public int getParkingSize (){
+        return this.parkingSize;
+    }
+
+    public void setParkingSize (String size) throws NumberFormatException{
```

Comment:
try formatting the classes using the Android Studio formatting tool.

`setParkingSize (String size)` should not have a space between method name and parenthesis.

it should be `setParkingSize(String size)`